sup3r/preprocessing/batch_handling.py

```diff
@@ -1112,14 +1453,123 @@ def __next__(self):
                 output_features_ind=self.output_features_ind,
                 training_features=self.training_features,
                 smoothing=self.smoothing,
-                smoothing_ignore=self.smoothing_ignore)
+                smoothing_ignore=self.smoothing_ignore,
+                model_mom1=self.model_mom1,
+                s_padding=self.s_padding,
+                t_padding=self.t_padding)
 
             self._i += 1
             return batch
         else:
             raise StopIteration
 
 
+class ValidationDataMom1SF(ValidationData):
```

Comment:
Maybe we could move some of this to a new file for conditional_moment_batch_handling